spec/tail_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'filewatch/tail'
+require 'stud/temporary'
+
+describe FileWatch::Tail do
+
+  let(:file_path) { Stud::Temporary.file.path }
+  let(:sincedb_path) { Stud::Temporary.file.path }
+
+  before :each do
+    Thread.new(subject) { sleep 0.5; subject.quit } # force the subscribe loop to exit
```

Comment:
Just a note here, keep in mind for each test defined not in a sub context, or sub describe, here, subject is the one you created in line 4 ( aka ```FileWatch::Tail.new```).